README.md

```diff
@@ -19,26 +17,28 @@ Import the module in `build.sc` using mill's `$ivy` import syntax, and extend `T
 ```scala
 // build.sc
 
-import $ivy.`io.github.davidgregory084::mill-tpolecat:0.2.0`
+import $ivy.`io.github.davidgregory084::mill-tpolecat_mill0.10:0.3.0`
```

Comment:
@joan38 Is this the correct syntax to use for importing this plugin now?

Reply:
That's correct. Thanks

Reply:
FYI, since Mill 0.9.10 the shorter version with two colons before version is also supported: 
```suggestion
import $ivy.`io.github.davidgregory084::mill-tpolecat::0.3.0`
```

Reply:
Nice I didn't know that.

Reply:
Oh that is really nice @lefou thanks for that!